AdblockPlusSafari/AdblockPlus.m

```diff
@@ -73,53 +82,42 @@ - (void)setActivated:(BOOL)activated
   [_adblockPlusDetails synchronize];
 }
 
-#pragma mark -
-
-- (NSString *)contentBlockerIdentifier
+- (void)setFilterlists:(NSDictionary<NSString *,NSDictionary<NSString *,NSObject *> *> *)filterlists
```

Comment:
I guess it's my lack of Objective-C knowledge, but I get the impression this setter is not actually used anywhere. Is it? If it is indeed not used, I would prefer to get rid of it - I'm not a fan of unused code.